xmlcommand/BESXMLInterface.cc

```diff
@@ -336,7 +335,7 @@ void BESXMLInterface::execute_data_request_plan()
             __LINE__);
 
         d_dhi_ptr->response_handler->execute(*d_dhi_ptr);
-
+        // @TODO Check RequestServiceTimer here?? Might be a good idea
```

Comment:
Would this be hard to do? 

It if so, then initial and date the comment.

Reply:
It would not be hard to. 
But I missed the comment before I merged.